### PR TITLE
reports: Add ksdme's GSoC report

### PR DESCRIPTION
**Pull request by ksdme**
https://deploy-preview-632--coala-projects.netlify.com/gsoc/2018/statusreport/ksdme

### CODE REVIEW

**Comment by ksdme** [comment]
Preview: https://deploy-preview-632--coala-projects.netlify.com/gsoc/2018/statusreport/ksdme

File: _reports/ksdme.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+
+  - 28:
+      - repo: coala-ls
+      - link: https://github.com/coala/coala-ls/pull/15/commits/6e095ca6c18813878766e9767d99a4bec10ec4a8
+      - details: >
+         perf-metrics.md: Update perf-metrics for Phase-2
+
+  - 29:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/pull/73/commits/52319c23e653850df148b284882fce799724c32c
```
Comment (jayvdb) [inline _reports/ksdme.md]: blocked on https://github.com/coala/coala-vs-code/pull/73 getting merged  -- those commit references will disappear on a rebase, so you can not use them.

File: _reports/ksdme.md [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+
+  - 3:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/pull/32/commits/bd12657c67b257ed4b0f498c14cab389f96cc91a
+      - details: >
+         [Bonding Period Work] langserver.py: Update to use jsonrpc
+
+  - 4:
+      - repo: coala-ls-gsoc
+      - link: https://github.com/ksdme/coala-ls-gsoc/commit/305a441742f109166bcae241e10b101ad5d908c9
```
Comment (jayvdb) [inline _reports/ksdme.md]: not suitable for the report

File: _reports/ksdme.md [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+      - details: >
+         coala_langserver: Drop codebase and tests
+
+  - 5:
+      - repo: coala-ls-gsoc
+      - link: https://github.com/ksdme/coala-ls-gsoc/commit/b1f5ae490fbafe06a14dcb34fa5578e2fbb118a7
```
Comment (jayvdb) [inline _reports/ksdme.md]: not suitable for the report

File: _reports/ksdme.md [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+
+  - 2:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/pull/171
+      - details: >
+         cEP-0028: Optimizing coala for language server
+
+  - 3:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/pull/32/commits/bd12657c67b257ed4b0f498c14cab389f96cc91a
```
Comment (jayvdb) [inline _reports/ksdme.md]: use https://github.com/coala/coala-vs-code/commit/bd12657c67b257ed4b0f498c14cab389f96cc91a to be consistent with the others.

File: _reports/ksdme.md [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+
+  - 1:
+      - repo: projects
+      - link: https://github.com/coala/projects/commit/10d706aa14325147caacd053e20c4fb77af56376
+      - details: >
+         [Bonding Period Work] coala_language_server.md: Update project details
+
+  - 2:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/pull/171
```
Comment (jayvdb) [inline _reports/ksdme.md]: merged as https://github.com/coala/cEPs/commit/07a63ec9  -- use short shas to keep line lengths under 80

File: _reports/ksdme.md [hunk 6]
```diff
@@ -0,0 +1,444 @@
...
+         Fix misconfigured endpoints
+  - 5:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/pull/32/commits/bd12657c67b257ed4b0f498c14cab389f96cc91a
+      - details: >
+         [Bonding Period Work] langserver.py: Update to use jsonrpc
+
+  - 6:
+      - repo: coala-ls-gsoc
+      - link: https://github.com/ksdme/coala-ls-gsoc/commit/305a441742f109166bcae241e10b101ad5d908c9
```
Comment (jayvdb) [inline _reports/ksdme.md]: can stay in your tarball, but doesnt belong in your activity repo.

File: _reports/ksdme.md [hunk 7]
```diff
@@ -0,0 +1,444 @@
...
+      - details: >
+         coala_langserver: Drop codebase and tests
+
+  - 7:
+      - repo: coala-ls-gsoc
+      - link: https://github.com/ksdme/coala-ls-gsoc/commit/b1f5ae490fbafe06a14dcb34fa5578e2fbb118a7
```
Comment (jayvdb) [inline _reports/ksdme.md]: can stay in your tarball, but doesnt belong in your activity repo.

File: _reports/ksdme.md [hunk 8]
```diff
@@ -0,0 +1,444 @@
...
+
+  - 34:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/commit/a0b59d36c33ae537b33ffd01d0f1a383ce1cdd57
+      - details: >
+         README.md: Update README.md
+
+  - 35:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/commit/2618f470e509a129e05f29ebf6cd9a7f300e9867
```
Comment (jayvdb) [inline _reports/ksdme.md]: can stay in your patches, but doesnt belong in the activity report

File: _reports/ksdme.md [hunk 9]
```diff
@@ -0,0 +1,444 @@
...
+      - details: >
+         Clean Repository
+
+  - 36:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/commit/9a9ace96b7bb498d6521f1e2b8944290ed3d331f
```
Comment (jayvdb) [inline _reports/ksdme.md]: can stay in your patches, but doesnt belong in the activity report

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/632/commits/62993d9179899888e6e7d62b8c2c95eb901e6aef

**Comment by jayvdb** [comment]
needs rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**